examples/audio/soundtrack.rs

```diff
@@ -0,0 +1,154 @@
+//! This example illustrates how to load and play different soundtracks
+
+use bevy::prelude::*;
+
+fn main() {
+    App::new()
+        .add_plugins(DefaultPlugins)
+        .add_systems(Startup, setup)
+        .add_systems(Update, (cycle_game_state, fade_in, fade_out))
+        .add_systems(Update, change_track.run_if(resource_changed::<GameState>))
```

Comment:
IMO this run condition is less clear than just checking `.is_changed` inside of `change_track`.

Reply:
Mildly faster for now, but that's really not worth optimizing. Especially not in a learning example.